plone-5/Makefile

```diff
@@ -7,6 +7,11 @@ all: build
 build:
 	docker build --build-arg POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) --tag plone .
 
+.PHONY: test
```

Comment:
this is still broke for me, same issue as other pr.

Reply:
doesn't surprise me there's issues, it felt such a hacky way of doing things 😂 Ill take another look